src/mod/externals/FlagWork_Enums.h

```diff
@@ -3241,6 +3246,10 @@ enum class FlagWork_Work : int32_t {
 
     // Luminescent Works
     WK_LAST_BATTLE_TURN_COUNTER = 449,
+    WK_GAME_MODE = 487,
+    WK_RANDOM_TEAMS = 488,
+    WK_TEMP_GAME_MODE = 489,
+    WK_TEMP_RANDOM_TEAMS = 490,
```

Comment:
Works 488-490 are already used in scripts (we haven't put stuff used in scripts into this enum yet)
You should be able to use 491-493, with any future ones ideally starting at 501 since those are from extended save and for sure unused